src/test/shell/integration/minimal_jdk_test.sh

```diff
@@ -42,13 +42,13 @@ export BAZEL_SUFFIX="_jdk_minimal"
 source "$(rlocation "io_bazel/src/test/shell/integration_test_setup.sh")" \
   || { echo "integration_test_setup.sh not found!" >&2; exit 1; }
 
-# Bazel's install base is < 275MB with minimal JDK and > 300MB with an all
+# Bazel's install base is < 280MB with minimal JDK and > 300MB with an all
 # modules JDK.
-function test_size_less_than_275MB() {
+function test_size_less_than_280MB() {
   bazel info
   ib=$(bazel info install_base)
   size=$(du -s "$ib" | cut -d\	 -f1)
-  maxsize=$((1024*275))
+  maxsize=$((1024*280))
```

Comment:
How much is it exactly after your change?

Reply:
It was failing on darwin: https://buildkite.com/bazel/bazel-bazel-github-presubmit/builds/6710#fec2a5d4-7cf4-47a6-bc19-215c8fd965b0 : 

`test_size_less_than_275MB FAILED: Size of install_base is 281984 kB, expected it to be less than 281600 kB.`

I may later try to check after/before on Mac.

Reply:
If you sync now, this change did already happen

Reply:
Indeed there was commit https://github.com/bazelbuild/bazel/commit/2b028e48947e9484a6a7e1d505daffdb390b5174
Merged this change with current master.